ros/src/fsds_ros_bridge/src/fsds_ros_bridge_camera.cpp

```diff
@@ -96,6 +96,30 @@ cv::Mat manual_decode_depth(const ImageResponse& img_response)
     return mat;
 }
 
+float roundUp(float numToRound, float multiple) 
+{
+    assert(multiple);
+    return ((numToRound + multiple - 1) / multiple) * multiple;
+}
```

Comment:
I am slightly confused about the rounding - how exactly does it work? Can you provide an example?

Reply:
copied it from here ;) https://stackoverflow.com/a/9194117/2328729

Reply:
it rounds up to the nearest multiple of another number